features/product/managing_products/select_taxons_for_product.feature

```diff
@@ -10,6 +10,7 @@ Feature: Select taxon for a product
         And the store has a "T-Shirt Banana" configurable product
         And the store has a product "T-Shirt Batman"
         And I am logged in as an administrator
+        And I am using "English (GB)" locale for my panel
```

Comment:
Does it translate to `en_GB` correctly?

Edit: Nope, `English (United Kingdom)` should be used.